legacy/src/scss/_vanilla-overrides.scss

```diff
@@ -0,0 +1,9 @@
+@media screen and (max-width: $breakpoint-small) {
```

Comment:
I think we want this to be `$breakpoint-navigation-threshold` instead, otherwise it's still broken in the midpoint between small and navigation-threshold

Reply:
Thanks! Well spotted. I've pushed a fix.